### PR TITLE
Add ProcessHelper #100

### PR DESCRIPTION
**Pull request by JunTaoLuo**
#100

### CODE REVIEW

File: src/Microsoft.Extensions.ProcessHelper.Sources/project.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿{
+  "version": "1.0.0-*",
+  "shared": "*.cs",
+  "dependencies": {
+  },
+  "frameworks": {
+    "net451": { },
+    "netstandard1.4": {
+      "dependencies": {
+        "System.Diagnostics.Process": "4.1.0-*"
```
Comment (pranavkm) [inline src/Microsoft.Extensions.ProcessHelper.Sources/project.json]: Shouldn't need this. Probably adding a reference to `System.Runtime` should suffice.

Comment (Eilon) [inline src/Microsoft.Extensions.ProcessHelper.Sources/project.json]: I thought you don't need *any* of this in a .Sources package... when I made one the other day, nothing here mattered as long as there was one TFM that was compatible everywhere, and that would be `netstandard1.0` with no dependencies at all.

File: src/Microsoft.Extensions.ProcessHelper.Sources/project.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+﻿{
+  "version": "1.0.0-*",
+  "shared": "*.cs",
+  "dependencies": {
```
Comment (victorhurdugaci) [inline src/Microsoft.Extensions.ProcessHelper.Sources/project.json]: No dependencies => property not needed

**Comment by victorhurdugaci** [comment]
Sorry but `ProcessHelper` is a terrible name. Name it `Microsoft.Extensions.Process.Sources`

**Comment by JunTaoLuo** [comment]
@victorhurdugaci Terribly consistent with the other sources project in the repo. Since it's not a package we ship, I don't think it matters much.

**Comment by victorhurdugaci** [comment]
:shipit:

**Pull request merged**